addons/binding/org.openhab.binding.mcp23017/README.md

```diff
@@ -0,0 +1,83 @@
+# MCP23017 Binding
+
+This binding allows you to have native access for MCP23017 I/O expander on I2C bus.
+It was tested with Raspberry PI 2 and Raspberry PI 3, but probably should work with other devices supported by pi4j library.
+
+On Raspberry PI user on which openHAB is running (default user name is "openhab") needs to be added to groups "i2c" and  "gpio".
+
+## Supported Things
+
+This binding supports one thing type:
+
+mcp23017 - which is a mcp23017 chip connected to a I2C bus on specified HEX address and bus number 
+
+## Discovery
+
+Discovery is not possible.
+
+## Binding Configuration
+
+No binding configuration required.
+
+## Thing Configuration
+
+* Required configuration for mcp23017 thing:
+
+    address - MCP23017 I2C bus address. On Raspberry PI it can be checked as a result of command: "i2cdetect -y 1". Value should be set in HEX.
+        Default value is "20"
+    bus_number - a bus number to which mcp23017 is connected. On RPI2 and RPI3 it will be "1", on RPI1 it will be "0".
+        Default value is "1"    
+
+## Channels
+
+mcp23017 supports 16 channels in 2 groups:
+
+ | Group | Channels | Additional parameters|
```

Comment:
Could you prettify this table (use e.g. https://www.tablesgenerator.com/markdown_tables)?